Tools/cases_generator/README.md

```diff
@@ -0,0 +1,33 @@
+# Tooling to generate interpreters
```

Comment:
I didn't review any of the files in this directory for correctness... the output smells okay to me, I'm not a parsing expert, and I'm sure the code will change in the near future. :)